### PR TITLE
Resolve bugs related to account creation and token balances

### LINKED ISSUE #5025: No randomness applied on the word selection in ConfirmPassphrase
### Expected behavior

As a user i need to select 2 random words for my passphrase at a random position of 24 words

### Actual behavior

A user always needs to fill in the 3rd and 10th word in that recovery passphrase

image
image

### Steps to reproduce

- Create an account on the latest beta version
- Select "add new account"

### Which version(s) does this affect? (Environment, OS, etc...)

MacOS

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5025, #5023, #5022, #5020, #5019

### How was it solved?

- Ensure randomness in confirm passphrase step
- Update tooltip text
- Handle send maximum amount
- Resolve account trash icon display

### How was it tested?

Visually

### CODE REVIEW

**Comment by ikem-legend** [comment]
There's a significant gap of about 32px between the "Continue to set password" and "Go back" buttons. I believe it should be a bit less than that

Screenshot 2023-05-30 at 15 37 40

File: src/modules/token/fungible/components/SendForm/form.test.js [hunk 1]
```diff
@@ -334,10 +337,10 @@ describe('Form', () => {
 
     it('Should allow to send 0 LSK amount', () => {
       const wrapper = mountWithQueryClient(Form, props);
-      const receipientEvt = {
+      const receipentEvt = {
```
Comment (ikem-legend) [inline src/modules/token/fungible/components/SendForm/form.test.js]: Typo in `receipentEvt`

File: src/utils/paperWallet.js [hunk 2]
```diff
@@ -92,10 +94,12 @@ class PaperWallet {
     return this;
   }
 
+  // eslint-disable-next-line max-statements
   renderAccount() {
-    const { t, passphrase } = this.props;
+    const { t } = this.props;
     const textOptions = this.textOptions;
-    const address = extractAddressFromPassphrase(passphrase);
+    // const address = extractAddressFromPassphrase(passphrase);
```
Comment (ikem-legend) [inline src/utils/paperWallet.js]: Comment should be removed

**Review by ikem-legend — COMMENTED** [review]
Just a few minor changes left. #5020 isn't resolved yet

File: src/utils/paperWallet.js (hunk 2 above)
Comment (eniolam1000752) [reply src/utils/paperWallet.js]: resolved

File: src/modules/token/fungible/components/SendForm/form.test.js (hunk 1 above)
Comment (eniolam1000752) [reply src/modules/token/fungible/components/SendForm/form.test.js]: resolved

**Comment by eniolam1000752** [comment]
> Just a few minor changes left. #5020 isn't resolved yet

I think moving this issue to a separate PR would be better so that it does not drag this PR for too long @ManuGowda

**Comment by eniolam1000752** [comment]
Would be removing #5020 from this PR

**Review by eniolam1000752 — DISMISSED** [review]

**Review by eniolam1000752 — DISMISSED** [review]

**Review by ikem-legend — DISMISSED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**